### PR TITLE
[27.x] Backports

### PR DESCRIPTION
**Pull request by fanquake**
Backports:
* https://github.com/bitcoin/bitcoin/pull/29691
* https://github.com/bitcoin/bitcoin/pull/29747
* https://github.com/bitcoin/bitcoin/pull/29776
* https://github.com/bitcoin/bitcoin/pull/29853
* https://github.com/bitcoin/bitcoin/pull/29856
* https://github.com/bitcoin/bitcoin/pull/29859
* https://github.com/bitcoin/bitcoin/pull/29869
* https://github.com/bitcoin/bitcoin/pull/29870
* https://github.com/bitcoin/bitcoin/pull/29886
* https://github.com/bitcoin/bitcoin/pull/29892
* https://github.com/bitcoin/bitcoin/pull/29934
* https://github.com/bitcoin/bitcoin/pull/29985

### CODE REVIEW

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -254,6 +256,7 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/fast_fixed_dtoa_no_optimize.patch && \
```
Comment (willcl-ark) [inline depends/packages/qt.mk:256]: In: a995902d604c701be4f46087057b907de9a0ecca

This also appears to pull in fast_fixed_dta_no_optimise.patch and I think that may not be intentional based on the commit it's rebasing?:

image

**Review by willcl-ark — COMMENTED** [review]
Left one question, looks good otherwise

File: depends/packages/qt.mk (hunk 1 above)
Comment (willcl-ark) [reply depends/packages/qt.mk:256]: Sorry for the noise, this is just my script misbehaving.
Script confused as fast_fixed_dtoa_no_dtoa_no_optimse was removed from master in the interim which confused the diffs.

**Review by willcl-ark — APPROVED** [review]
ACK 46890065e59875dd3d736d04f427e410bbd8faed

Without my earlier misunderstanding, this all looks good to me.

File: doc/release-notes.md [hunk 2]
```diff
@@ -34,184 +34,55 @@ Core should also work on most other Unix-like systems but is not as
 frequently tested on them. It is not recommended to use Bitcoin Core on
 unsupported systems.
 
-Notable changes
-===============
```
Comment (stickies-v) [inline doc/release-notes.md:38]: I think we usually keep this header?

File: doc/release-notes.md [hunk 3]
```diff
@@ -34,184 +34,55 @@ Core should also work on most other Unix-like systems but is not as
...
+- #29870 rpc: Reword SighashFromStr error message
 
-  In the future, libbitcoinkernel will provide a much more useful API that is
-  aware of the UTXO set, and therefore be able to fully validate transactions and
-  blocks. (#29189)
+### Index
 
-mempool.dat compatibility
--------------------------
+- #29767 Fix #29767, set m_synced = true after Commit()
```
Comment (stickies-v) [inline doc/release-notes.md:48]:
```suggestion
- #29776 Fix #29767, set m_synced = true after Commit()
```

Comment (stickies-v) [reply doc/release-notes.md:48]: Also, this is the commit message and not the PR title but this is a bit more descriptive so makes sense.

**Review by stickies-v — APPROVED** [review]
ACK 46890065e59875dd3d736d04f427e410bbd8faed

All backports clean except for 0fcceefe22532dc6389a95d2e058599e9496003b because of #28955, but the backport looks good to me. CI failure looks unrelated.

File: doc/release-notes.md (hunk 3 above)
Comment (fanquake) [reply doc/release-notes.md]: Thanks, addressed both.

**Comment by willcl-ark** [comment]
reACK bd5860bc7a892c6bcffe313246dd6b81b973b9c6

**Review by stickies-v — APPROVED** [review]
re-ACK bd5860bc7a892c6bcffe313246dd6b81b973b9c6

**Comment by hebasto** [comment]
I suggest to add https://github.com/bitcoin-core/gui/pull/813.

**Review by sedited — APPROVED** [review]
ACK bd5860bc7a892c6bcffe313246dd6b81b973b9c6

**Comment by fanquake** [comment]
> I suggest to add https://github.com/bitcoin-core/gui/pull/812 and https://github.com/bitcoin-core/gui/pull/813.

I'll put these into the next PR, that will likely also do release prep.

**Pull request merged**